### ISSUE TITLE
DRY out vignette headers

### ISSUE THREAD

**Issue by pearsonca** [epinowcast/epinowcast#424]
As follow up to team authorship PR:

Seems like this approach might be ripe for DRYing out ala https://stackoverflow.com/questions/39885363/importing-common-yaml-in-rstudio-knitr-document

_Originally posted by @pearsonca in https://github.com/epinowcast/epinowcast/issues/421#issuecomment-1876067791_

**Comment by seabbs** [comment]
When you get a chance could you flesh this out a bit, label and add to the `0.5.0` release (next infra release)? I found it a bit hard to parse without clicking through so I imagine might be a struggle for anyone looking at this and thinking about wantin go contribute.

**Comment by pearsonca** [comment]
nice work, @seabbs-bot 🥇
